cdn.py

```diff
@@ -25,6 +27,25 @@ def fastly_purge(*args):
             run("curl -s -X PURGE {0}{1} | grep 'ok'".format(hostname, govuk_path.strip()))
 
 
+@task
+@runs_once
+def purge_all_document_attachments(arg):
+    "Purge each of the attachments for a specific document."
+    if env.environment == 'production':
+        hostname = 'https://www.gov.uk/api/content/'
+    elif env.environment == 'staging':
+        hostname = 'https://www-origin.staging.publishing.service.gov.uk/api/content/'
+
+    response = requests.get("{0}{1}".format(hostname, arg))
+    if response.status_code == 200:
+        try:
+            attachments = response.json()['details']['attachments']
+            attachment_urls = [urlparse(attachment['url']).path for attachment in attachments]
+            execute(fastly_purge, *attachment_urls)
```

Comment:
Does this log the URLs that are being purged? It seems like a useful thing to know, so you can check it's purging the URLs you expect.